lib/client/terminal/terminal_windows.go

```diff
@@ -272,6 +272,10 @@ func (t *Terminal) Stderr() io.Writer {
 	return t.stderr
 }
 
+func (t *Terminal) Clear() error {
+	return nil
```

Comment:
Does this need to work on Windows too?

Reply:
#6107 only asks for it to work in the web terminal.

Reply:
I actually don't know the answer to this, but would this windows terminal not show up in the web UI as well? Is there any reason we shouldn't make it work on Windows?

Reply:
We should support all terminals. The Web UI was requested in the ticket, but after talking with the customer, it's a FedRAMP requirement for all terminals.

If the escape code approach works for the Web UI (as in xterm.js understands it) we should just make the changes in `lib/client` and let all terminals automatically pick it up.

Reply:
xtermjs does understand it

Reply:
I've tested the escape code in Windows. Command line and Powershell did not work (although [wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code#DOS,_OS/2,_and_Windows) suggests that they could with some fiddling), but Windows Terminal does.

Reply:
@timothyb89 What does `tsh.exe` support? Also, any hints on how @atburke could handle this for Windows?

Reply:
tsh does enable ANSI processing on Windows, so assuming `Terminal.Close()` hasn't yet been called (which appears to be the case) I'd expect the usual escape sequence to just work. Microsoft recommends the same: https://docs.microsoft.com/en-us/windows/console/clearing-the-screen

There are other fallback methods involving old Win32 APIs if we _really_ have to, but hopefully we can get the ANSI sequence working.

(Slight possible caveat that the ANSI sequences might not work on very early versions of Windows 10, as in v1609 if not earlier. Hopefully we don't need to worry about them w.r.t compliance issues?)

Reply:
I agree, we should not fall back to Win32 or support very early version of Windows 10.

@atburke What version of Windows were you using?

Reply:
This was on Windows 10 version 1809.

Reply:
I just copied the impl over from `terminal_unix.go` to `terminal_windows.go` and it cleared my cmd.exe just fine (on Win10 20H2)